meta-genivi-dev/recipes-support/keyutils/keyutils_%.bbappend

```diff
@@ -0,0 +1,2 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
+SRC_URI_append = " file://keyutils-arm-remove-m32-m64.patch"
```

Comment:
> For some PR builds we get this error:
Patch keyutils-arm-remove-m32-m64.patch is already applied; check your series file

@gunnarx You're right. I missed "aarch64" override. This is only for 64bit.